internal/runner/service_test.go

```diff
@@ -347,6 +347,9 @@ func Test_runnerService(t *testing.T) {
 	})
 
 	t.Run("Input", func(t *testing.T) {
+		if os.Getenv("RUN_FLAKY") == "" {
```

Comment:
This is not the way to go. We should make such tests more robust instead of marking them as flaky. Otherwise, they will never be fixed. I am not sure if this test is any more flaky than other ones.

Reply:
> This is not the way to go. We should make such tests more robust instead of marking them as flaky. Otherwise, they will never be fixed. I am not sure if this test is any more flaky than other ones.

@adambabik I don't disagree with you on the principle but it's beyond Jeremy's involvement to fix tests in the existing test suite beyond a reasonable effort (which he's already making). I think it's fair marking them so we can continue to target them while it removes ambiguity whether test failures are connected or disconnect to unrelated changes.

Reply:
I noticed that now all tests passed, so that single test isn't more flaky than others. But it's a good point that if any test is blocking a merge then this approach seems good, or commenting it out entirely plus creating an issue is also fine.

Unless this is also a problem locally then I would be happy to discuss it on Discord in details, @jlewi.

Reply:
@jlewi we can introduce this, but I suggest changing it slightly:

```suggestion
		if skip, err := strconv.ParseBool(os.Getenv("SKIP_FLAKY")); err == nil && skip {
```

It will be more explicit what's the intention: if the `SKIP_FLAKY` env is present and set to [truthy value](https://pkg.go.dev/strconv#ParseBool) then skip this flaky test.